library/std/src/os/unix/io/mod.rs

```diff
@@ -6,7 +6,7 @@
 //!
 //! This module provides three types for representing file descriptors,
 //! with different ownership properties: raw, borrowed, and owned, which are
-//! analogous to types used for representing pointers:
+//! analogous to types used for representing pointers. These types realize the Unix version of [I/O safety].
```

Comment:
"realize" might suggest that these types are the extent of I/O safety. However, the intent is that the rules be respected even for code that calls FFI functions using file descriptors, without using these types at all.

Reply:
I can make it say "reflect" instead, which is also the verb I used in the `io` docs ("The platform-specific parts of the Rust standard library expose types that reflect these concepts")